src/malli/generator.cljc

```diff
@@ -158,6 +158,18 @@
 ;; generators
 ;;
 
+(defn- qualified-keyword-generator [schema]
```

Comment:
could this be `-qualified-ident-gen` with +3 args? The two functions are basically the same.

Reply:
Sure. After I created this PR I found that you have some code before `generators` comment. I was confused initially what that meant to I've put my generator below it. Because it... marks the generators. 
I'll do the change.
I first wanted to make sure this change makes sense.

Reply:
I did some code reuse commits but I don't like where it went. I'm not sure if it's more readable than before.
I prefer the version from commit [817652f](https://github.com/metosin/malli/pull/643/commits/817652f5d4ce588af38e46fab4b969587fe46f1f).